### ISSUE TITLE
RenderWindow store

### ISSUE THREAD

**Issue by Transliteration** [BaatleTeam/Tetris#14]
Определить где хранить и как передавать RenderWindow.

**Comment by Denmey** [comment]
А где он используется? Где не используется? Может, хранить его в классе GameController?
Отовсюду ли у нас видно объект GameController?

**Comment by makometr** [comment]
> А где он используется? Где не используется? Может, хранить его в классе GameController?
> Отовсюду ли у нас видно объект GameController?

Нет, нет, нет. GameController что-то наподобие тетриса  в себе. Он даже не должен догадываться от том, что его пытаются визуализировать. Объект gameController является privаte-членом класса GameScreen (на данный момент).

**Comment by makometr** [comment]
После принятия #19 предлагаю RenderWindow перенести туда (в ScreenManager)
